src/widget/form/settings/avform.cpp

```diff
@@ -232,6 +232,10 @@ void AVForm::updateVideoModes(int curIndex)
     {
         int i = iter->second;
         VideoMode mode = allVideoModes[i];
+        // don't use the same mode twice
```

Comment:
Just a small style fix. It's totally clear what the code does. No comment needed here. Can you just replace with empty line?  Thanks.